src/library_formatString.js

```diff
@@ -174,9 +174,6 @@ mergeInto(LibraryManager.library, {
             var signed = next == {{{ charCode('d') }}} || next == {{{ charCode('i') }}};
             argSize = argSize || 4;
             currArg = getNextArg('i' + (argSize * 8));
-#if PRECISE_I64_MATH
```

Comment:
I started like that, but the code is testing `if (typeof i64Math === 'object')` which is always false, so I removed this altogether because that condition will never be true. Since `src/long.js` is never emitted to the code, we do not have a `i64Math` object ever being generated, so the condition there never passes.

Reply:
I see, sounds good then.